db/seeds.rb

```diff
@@ -1,5 +1,17 @@
 # frozen_string_literal: true
-User.where(email: 'test@test.com').first_or_create(password: 'password',
-                                                   role: :admin)
-User.where(email: 'user@test.com').first_or_create(password: 'password',
-                                                   role: :reporter)
+User.where(email: 'admin@test.com').first_or_create(name: 'Administrator',
```

Comment:
just a suggestion ... but it can help with future development/testing ... add "any_login" gem, and you will be able to quickly login as any user during development (and show it to other devs)